Makefile

```diff
@@ -29,7 +29,12 @@ RELEASE_BUCKET ?= $(PROJECT)
 
 VERSION_PACKAGE := $(REPOPATH)/pkg/version
 
+# If the architecture is not amd64, only create the linux binary
```

Comment:
Well given that the only architecture we are testing at the moment that is not `amd64` is `arm64` then at the moment yes.

The reason I did this is because the compilation fails when compiling the `darwin` platform binary on `arm64` and I believe this is because `darwin` does not support arm64 at present.

Here is the Travis CI when it was attempting to build the binary:
https://travis-ci.org/github/arg2009/container-structure-test/jobs/692949887

It did not affect the CI test at all because the `darwin` binary was not being used in the tests anyway but I just thought it we should restrict the compilation of the binary to architectures that support cross platform, which is currently only `amd64`.

I can revert this if you would like, it just means that the Travis CI log will show an error during the cross platform compilation phase.

Reply:
ahh ok that makes sense. I suppose this is fine to keep, it feels weird to be special casing an architecture when compiling as a result of another quirk of our process, but seeing as we have lots of quirks in our release process anyway, let's roll with it